services/QuillLMS/app/serializers/progress_reports/standards/standard_serializer.rb

```diff
@@ -15,10 +15,17 @@ class ProgressReports::Standards::StandardSerializer < ActiveModel::Serializer
              :timespent,
              :average_score,
              :standard_students_href,
-             :mastery_status
+             :mastery_status,
+             :is_evidence
+
+  # rubocop:disable Naming/PredicateName
```

Comment:
I believe this rubocop exception is justified, since `is_evidence` is more intelligible on the front end, versus, say, `evidence_activity`